plugins/bufbuild/connect-kotlin/v0.1.8/buf.plugin.yaml

```diff
@@ -4,6 +4,7 @@ plugin_version: v0.1.8
 source_url: https://github.com/bufbuild/connect-kotlin
 description: Idiomatic gRPC & Connect RPCs for Kotlin.
 deps:
+  # v23.4 revision updated 20230724
```

Comment:
Can you remind me, why we need this line?

Reply:
Ah right, to bump this so it depends on the newer patched revision. nvm

Reply:
This is needed to trigger a re-push of the plugin (with different buf.plugin.yaml), which when installed will automatically pull in the latest revision. Right now the previous version is installed and pins to the previous revision of the dependency.